src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -43,7 +44,10 @@ function getSentryIntegrations() {
     // TODO(ts): The type returned by SentryRRWeb seems to be somewhat
     // incompatible. It's a newer plugin, so this can be expected, but we
     // should fix.
-    integrations.push(new SentryRRWeb() as any);
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
I think it'll be useful to have a env variable to enable it in dev just in case. Maybe an opt-in.

Reply:
maybe like `SENTRY_RRWEB=1 sentry devserver --workers`